src/main/scala/mesosphere/marathon/core/launcher/TaskLauncher.scala

```diff
@@ -0,0 +1,8 @@
+package mesosphere.marathon.core.launcher
+
+import org.apache.mesos.Protos.{ OfferID, TaskInfo }
+
+trait TaskLauncher {
```

Comment:
Even if this trait is pretty self explanatory, add one sentence regarding responsibility.